lib/widgets/compose_box.dart

```diff
@@ -630,11 +641,11 @@ class _StreamContentInputState extends State<_StreamContentInput> {
         // The chosen topic can't be sent to, so don't show it.
         return null;
       }
-      if (!widget.controller.contentFocusNode.hasFocus) {
-        // Do not fall back to a vacuous topic unless the user explicitly chooses
-        // to do so (by skipping topic input and moving focus to content input),
-        // so that the user is not encouraged to use vacuous topic when they
-        // have not interacted with the inputs at all.
+      if (widget.controller.topicInteractionStatus.value !=
+          ComposeTopicInteractionStatus.hasChosen) {
+        // Do not fall back to a vacuous topic unless the user explicitly
```

Comment:
compose: Change content input hint text statefully
    
    Before, the content input shows the "#stream > topic" hint text
    as long as it has focus, and set the hint text to "#stream" when it
    loses focus.
    
    Now, the content input still shows "#stream > topic" when it gains
    focus, except that it will keep showing it even after losing focus,
    until the user moves focus to the topic input.

This change only affects the case where the topic is vacuous, right? As written, the commit message sounds like it applies to an arbitrary topic.

I think replacing "topic" with "general chat" in the examples would go most of the way to clearing that up. Also add a condition in the first sentence (the "Before, …").